### PR TITLE
spellcheck and comments fix

### PR DESCRIPTION
**Pull request by volekerb**
### Problem
According to English language and
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
we have multiple small issues.
Decided to walk through them and fix in one shot.
...

### Solution
This patchset hopefully fixes all the possible issues in current codebase.
...

### CODE REVIEW

**Comment by volekerb** [comment]
+ Coverage   48.27%   48.63%   +0.35% 
that is fun

File: pkg/vm/compiler/codegen.go [hunk 1]
```diff
@@ -805,7 +805,7 @@ func CodeGen(info *buildInfo) (*bytes.Buffer, error) {
 			for _, decl := range f.Decls {
 				switch n := decl.(type) {
 				case *ast.FuncDecl:
-					// Dont convert the function if its not used. This will save a lot
+					// Does not convert the function if its not used. This will save a lot
```
Comment (roman-khimov) [inline pkg/vm/compiler/codegen.go]: `Don't convert the function if it's not used`?

File: pkg/vm/compiler/func_scope.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ type funcScope struct {
 	// return value to the stack size.
 	voidCalls map[*ast.CallExpr]bool
 
-	// local variable counter
+	// Local variable counter
```
Comment (roman-khimov) [inline pkg/vm/compiler/func_scope.go]: dot?

File: pkg/util/uint160.go [hunk 3]
```diff
@@ -49,7 +49,7 @@ func (u Uint160) String() string {
 	return hex.EncodeToString(u.Bytes())
 }
 
-// ReverseString is the same as String, but returnes an inversed representation.
+// ReverseString is the same as String, but returns an inverses representation.
```
Comment (roman-khimov) [inline pkg/util/uint160.go]: `returns an inversed`

File: pkg/core/blockchain.go [hunk 4]
```diff
@@ -20,7 +20,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// tuning parameters
+// tuning parameters.
```
Comment (roman-khimov) [inline pkg/core/blockchain.go]: `Tuning`?

File: pkg/core/interop_system.go [hunk 5]
```diff
@@ -340,15 +340,15 @@ func (ic *interopContext) runtimeCheckWitness(v *vm.VM) error {
 	return nil
 }
 
-// runtimeNotify should pass stack item to the notify plugin to handle it, but
+// runtimeNotify passes stack item to the notify plugin to handle it, but
```
Comment (roman-khimov) [inline pkg/core/interop_system.go]: Don't think so.

File: pkg/core/storage/memory_store.go [hunk 6]
```diff
@@ -14,7 +14,7 @@ type MemoryStore struct {
 	del map[string]bool
 }
 
-// MemoryBatch a in-memory batch compatible with MemoryStore.
+// MemoryBatch an in-memory batch compatible with MemoryStore.
```
Comment (roman-khimov) [inline pkg/core/storage/memory_store.go]: `is`?

File: pkg/core/interop_system.go (hunk 5 above)
Comment (volekerb) [reply pkg/core/interop_system.go]: good catch!

File: pkg/util/uint160.go (hunk 3 above)
Comment (roman-khimov) [reply pkg/util/uint160.go]: Still there.

**Pull request merged**